components/CoursePage/CoursePage.jsx

```diff
@@ -1,128 +1,153 @@
-import React, { useEffect, useState } from 'react'
-import CourseCard from '../CourseCard/CourseCard'
-import Carousel from './Subject-carousel'
-import { useDispatch, useSelector } from 'react-redux'
-import { loadAllCourses } from '../../store/courses/thunks'
-import { getAllCourses } from '../../store/courses/selectors'
-import RegistrationModal from '../Form/Registration'
-import LoginModal from '../Form/Login'
-import { getIsAuth } from '../../store/auth/selectors'
-import { useTranslation } from 'next-i18next'
+import React, { useEffect, useState } from "react";
+import CourseCard from "../CourseCard/CourseCard";
+import Carousel from "./Subject-carousel";
+import { useDispatch, useSelector } from "react-redux";
+import { loadAllCourses } from "../../store/courses/thunks";
+import { getAllCourses } from "../../store/courses/selectors";
+import RegistrationModal from "../Form/Registration";
+import LoginModal from "../Form/Login";
+import { getIsAuth } from "../../store/auth/selectors";
+import { useTranslation } from "next-i18next";
+import { Select, Input } from "antd";
+const { Option } = Select;
+const { Search } = Input;
 
 const CoursePage = () => {
-  const { t } = useTranslation('programs')
+  const { t } = useTranslation("programs");
 
-  const [loginModalShow, setLoginModalShow] = useState(false)
-  const [registrationModalShow, setRegistrationModalShow] = useState(false)
-  const isAuth = useSelector(getIsAuth)
+  const [loginModalShow, setLoginModalShow] = useState(false);
+  const [registrationModalShow, setRegistrationModalShow] = useState(false);
+  const isAuth = useSelector(getIsAuth);
 
   const switchModals = () => {
-    setLoginModalShow((prev) => !prev)
-    setRegistrationModalShow((prev) => !prev)
-  }
+    setLoginModalShow((prev) => !prev);
+    setRegistrationModalShow((prev) => !prev);
+  };
 
-  const dispatch = useDispatch()
-  const courses = useSelector(getAllCourses)
+  const dispatch = useDispatch();
+  const courses = useSelector(getAllCourses);
   useEffect(() => {
-    dispatch(loadAllCourses(1, ''))
-  }, [])
+    dispatch(loadAllCourses(1, ""));
+  }, []);
 
   const coursesBlock = courses.map((course, index) => {
-    const countBlock = index % 3 === 0 ? 8 : 4
+    const countBlock = index % 3 === 0 ? 8 : 4;
     return (
       <div className={`col-12 col-md-${countBlock}`} key={course._id}>
-        <CourseCard course={course}/>
+        <CourseCard course={course} />
       </div>
-    )
-  })
+    );
+  });
+
+  const categories = [
+    { label: t("category") },
```

Comment:
You don't need to have array of objects, make just array `[t("category"), t("math")]`